low-level/framework/target/enclosure.py

```diff
@@ -30,7 +30,7 @@
 class StorageEnclosure(object):
 
     ENCL_FAMILY = "enclosure-family"
-    LDR_R1_ENCL = "Realstor Storage_enclosure"
+    LDR_R2_ENCL = "Realstor Storage_enclosure"
```

Comment:
just keep LDR_ENCL

Reply:
Reverted. LDR_R1 renaming will be addressed in a separate ticket. 
Now I renamed only in few places that are blocked this current PR work.